install/kubernetes/values.yaml

```diff
@@ -160,6 +162,7 @@ tetragonOperator:
     suffix: ""
   # Skip CRD creation.
   skipCRDCreation: false
+  skipTetragonPodCRD: false
```

Comment:
```suggestion
```
let's just use `tetragonPod.enabled` as the feature flag.

Reply:
sure